CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# CODEOWNERS for autoreview assigning in github
```

Comment:
code owners file seems to have errors?

Reply:
They are not yet part of Zephyr org, and @carlescufi told me that once this PR is merged they will be invited to join automatically.